test/e2e/testkit/mocks/backend_deployment.go

```diff
@@ -16,13 +16,15 @@ const (
 	replicas           = 1
 	otelCollectorImage = "europe-docker.pkg.dev/kyma-project/prod/tpi/otel-collector:0.79.0-58a64bcf"
 	nginxImage         = "europe-docker.pkg.dev/kyma-project/prod/external/nginx:1.23.3"
+	fluentDImage       = "fluent/fluentd:v1.16-debian-1"
```

Comment:
We should copy this image to our own registry. I think you can just add the image [this list](https://github.com/kyma-project/test-infra/blob/7b90c757d6402345257bd43cdb376c1d98e639de/development/image-syncer/external-images.yaml).